RecoCTPPS/PixelLocal/src/RPixDetClusterizer.cc

```diff
@@ -0,0 +1,160 @@
+#include <iostream>
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "RecoCTPPS/PixelLocal/interface/RPixDetClusterizer.h"
+
+namespace {
+  constexpr int max16bits = 65535;
+  constexpr int maxCol = 155;
+  constexpr int maxRow = 159;
```

Comment:
```CTPPSPixelCluster::maxCol ``` have the same value, if it is logically the same, there should be no redefinition, use ```constexpr int maxCol = CTPPSPixelCluster::maxCol``` instead or even use that directly